composer.json

```diff
@@ -3,15 +3,35 @@
     "description": "Symfony bundle for information about your Messenger queues and transports",
     "type": "symfony-bundle",
     "require": {
+        "php": "^7.1.3",
         "symfony/framework-bundle": "^4.4|^5.0",
-        "symfony/messenger": "^4.4|^5.0"
+        "symfony/messenger": "^4.4|^5.0",
+        "symfony/twig-bundle": "^4.4|^5.0"
+    },
+    "require-dev": {
+        "roave/security-advisories": "dev-master",
```

Comment:
I'm not sure this package makes sense for a library - as packages are more about which is the lowest "allowed" package vs what's installed in the `.lock`file